assets/js/components/settings/SettingsAdmin.js

```diff
@@ -70,150 +71,97 @@ const SettingsAdmin = () => {
+								<Grid>
+									<Row>
+										<Cell size={ 12 }>
+											<h3 className="googlesitekit-heading-4 googlesitekit-settings-module__title">
 												{ __( 'Your site goals', 'google-site-kit' ) }
 											</h3>
 											<p>
 												{ __( 'Based on your responses, Site Kit will show you metrics and suggestions that are specific to your site to help you achieve your goals', 'google-site-kit' ) }
 											</p>
-										</div>
-									</div>
+										</Cell>
+									</Row>
+
 									<UserInputPreview goTo={ goTo } noFooter />
-								</div>
+								</Grid>
 							</div>
 						</Layout>
 					) }
 					{ ! isUserInputCompleted && (
 						<UserInputSettings isDismissable={ false } />
 					) }
-				</div>
+				</Cell>
 			) }
-			<div className="
-				mdc-layout-grid__cell
-				mdc-layout-grid__cell--span-12
-			">
+
+			<Cell size={ 12 }>
 				<Layout>
-					<div className="
-						googlesitekit-settings-module
-						googlesitekit-settings-module--active
-					">
-						<div className="mdc-layout-grid">
-							<div className="mdc-layout-grid__inner">
-								<div className="
-									mdc-layout-grid__cell
-									mdc-layout-grid__cell--span-6-desktop
-									mdc-layout-grid__cell--span-4-tablet
-									mdc-layout-grid__cell--span-4-phone
-								">
-									<h3 className="
-										googlesitekit-heading-4
-										googlesitekit-settings-module__title
-									">
+					<div className="googlesitekit-settings-module googlesitekit-settings-module--active">
+						<Grid>
+							<Row>
+								<Cell size={ 12 }>
+									<h3 className="googlesitekit-heading-4 googlesitekit-settings-module__title">
 										{ __( 'Plugin Status', 'google-site-kit' ) }
 									</h3>
-								</div>
-								<div className="
-									mdc-layout-grid__cell
-									mdc-layout-grid__cell--span-6-desktop
-									mdc-layout-grid__cell--span-4-tablet
-									mdc-layout-grid__cell--span-4-phone
-									mdc-layout-grid__cell--align-middle
-									mdc-layout-grid__cell--align-right-tablet
-								">
-								</div>
-								<div className="
-									mdc-layout-grid__cell
-									mdc-layout-grid__cell--span-12
-								">
+								</Cell>
+								<Cell size={ 12 }>
 									<div className="googlesitekit-settings-module__meta-items">
 										<p className="googlesitekit-settings-module__status">
 											{ __( 'Site Kit is connected', 'google-site-kit' ) }
-											<span className="
-												googlesitekit-settings-module__status-icon
-												googlesitekit-settings-module__status-icon--connected
-											">
+											<span className="googlesitekit-settings-module__status-icon googlesitekit-settings-module__status-icon--connected">
 												<VisuallyHidden>
 													{ __( 'Connected', 'google-site-kit' ) }
 												</VisuallyHidden>
 											</span>
```

Comment:
A bit unrelated to the changes being made, but this icon doesn't need "alt" text like this and we should remove it.

The status is already read above ("Site Kit is connected"), so this text is just floating here without context and would result in a screen reader reading: "Site Kit is connected Connected".

A follow-up issue might be better, as I see this pattern in two files in the code.

Reply:
Filed #3763 🙂